README.md

```diff
@@ -1,38 +1,63 @@
 # Service Endpoint Definition
-Repository to share service endpoint definitions (SEDs) 
+This is the repository to share service endpoint definitions (SEDs). SEDs are secrets containing Service Specific information such as endpoints and credentials. These secrets can be bound to running kubernetes workolads so that they are able to connect to the service they represent.
```

Comment:
```suggestion
This is the repository to share service endpoint definitions (SEDs). SEDs are secrets containing service-specific information such as endpoints and credentials. These secrets can be bound to running Kubernetes workloads so that they can connect to the service they represent.
```